satpy/etc/readers/seviri_l2_bufr.yaml

```diff
@@ -58,6 +58,7 @@ datasets:
     key: 'latitude'
     resolution: [48006.450653072,9001.209497451]
     file_type: [seviri_l2_bufr_asr,seviri_l2_bufr_cla,seviri_l2_bufr_csr,seviri_l2_bufr_gii,seviri_l2_bufr_thu,seviri_l2_bufr_toz]
+    long_name: Latitude
```

Comment:
i'd like to see the order here as in the GRIB-yaml file, i.e. `name`, `long_name`, `standard_name`, and then the other keys. please move these up.